tests/integration/auth.spec.coffee

```diff
@@ -58,7 +58,12 @@ describe 'SDK authentication', ->
 					email: credentials.username,
 					password: 'NOT-THE-CORRECT-PASSWORD'
 
-				m.chai.expect(promise).to.be.rejectedWith('Unauthorized')
+				m.chai.expect(promise).to.be.rejected
+				.then (error) ->
+					m.chai.expect(error).to.have.property('statusCode')
+					.and.satisfy (statuCode) ->
+						statuCode == 401 or # 'Unauthorized'
+						statuCode == 429 # 'Too many requests in this time frame.'
```

Comment:
I don't think this is right - we shouldn't be hitting rate limits here, and if we are we should fix that properly in general, not just for this special case.

Reply:
Agreed, removing this commit for now. I guess that this only helped since I was using `.only` for this section of the tests. Maybe checking just that the promise was rejected is enough for this "fails to login with wrong password" test case.